src/VisualStudio/Core/Impl/SolutionExplorer/SymbolTree/RootSymbolTreeItemSourceProvider.cs

```diff
@@ -211,5 +213,29 @@ void OnItemPropertyChanged(object sender, PropertyChangedEventArgs e)
                 }
             }
         }
+
+        void AddToDictionary(string currentFilePath, RootSymbolTreeItemCollectionSource source)
```

Comment:
(feel free to ignore since this is impacting RPS)

You can just use

https://github.com/jasonmalinowski/roslyn/blob/ef33d325804d8743dfed9c9a1a4eec249a8a6ab9/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/IDictionaryExtensions.cs?plain=1#L63C1-L74C6

Reply:
Good to know for the future, thanks.